force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +110,22 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
-            // get individual characters from the string - hint use substring or split
+        Integer result = 0;
+        str = str.toLowerCase();
+        List<String> eachLetterList = str.split('');
+        // get individual characters from the string - hint use substring or split
+        for (String eachLetter : eachLetterList) {
             // check if the character is a vowel
+            if(eachLetter == 'a' || eachLetter == 'e' || eachLetter == 'i' || eachLetter == 'o' || eachLetter == 'u' ){
```

Comment:
you can probably simplify this to just a Set of every vowel, and check if the set contains the letter.